SplunkAppForWazuh/appserver/controllers/api.py

```diff
@@ -22,6 +22,43 @@
 from log import log
 
 
+# This function change the keys for "********" for don't show the password in the app 
+def cleanKeys(response): 

```

Comment:
Replace `# This function change the keys for "********" for don't show the password in the app 
` by `# Useful function for removing sensible keys`

Reply:
Please check this out and test it again. I just realized that this function is provoking `ERROR` logs. Thanks.